docs/settings/alert-action-settings.asciidoc

```diff
@@ -51,6 +54,23 @@ You can configure the following settings in the `kibana.yml` file.
   +
   Disabled action types will not appear as an option when creating new connectors, but existing connectors and actions of that type will remain in {kib} and will not function.
 
+| `xpack.actions.preconfigured`
+  | Specifies preconfigured action IDs and configs. It defaults to {}.
+
+| `xpack.actions.proxyUrl` {ess-icon}
+  | Specifies the proxy URL to use, if using a proxy for actions. By default is no proxy used.
+
+| `xpack.actions.proxyHeader` {ess-icon}
+  | Specifies HTTP headers for proxy, if using a proxy for actions.
+
+| `xpack.actions.proxyRejectUnauthorizedCertificates` {ess-icon}
```

Comment:
Can we add a line break to this config name so it doesn't make this column width so wide?